pkgs/development/python-modules/chromaprint/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, python3
```

Comment:
python modules are not given a specific python version as they can be built against multiple versions. Instead guard on versions.